spanner/transaction.go

```diff
@@ -1676,7 +1681,7 @@ func (t *writeOnlyTransaction) applyAtLeastOnce(ctx context.Context, ms ...*Muta
 					return ToSpannerError(err)
 				}
 			}
-			res, err := sh.getClient().Commit(contextWithOutgoingMetadata(ctx, sh.getMetadata()), &sppb.CommitRequest{
+			res, err := sh.getClient().Commit(contextWithOutgoingMetadata(ctx, sh.getMetadata(), true), &sppb.CommitRequest{
```

Comment:
Shouldn't this just use the client setting for `disableRouteToLeader`?

If not: It is a read/write transaction, so shouldn't the default then be `false`?

Reply:
Thanks, fixed